spec/hobbit_spec.rb

```diff
@@ -58,21 +58,36 @@
     expect(hobbit.adult?).to be true
   end
 
-  xit 'is old at the age of 101' do
-    # create a hobbit
+  it 'is old at the age of 101' do
+    hobbit = Hobbit.new('Stemmle')
+
```

Comment:
@Taamir-Ransome This test does not show the age of hobbit. We can see that the hobbit is celebrating birthdays, but this test should should assert the age to give us a warm fuzzy feeling that things are working properly...
```
hobbit = Hobbit.new('Stemmle')
expect(hobbit.age).to eq 0
expect(hobbit.old?)to be false
100.times { hobbit.celebrate_birthday }
expect(hobbit.old?)to be false
hobbit.celebrate_birthdate
expect(hobbit.age).to eq 101
expect(hobbit.old?)to be true
```